.changeset/eight-pots-remember.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': minor
```

Comment:
Just needs to be a `patch` release for now as we're below `1.0.0` in the `scaffolder-backend`. When it goes to GA then new features are minor :)

Reply:
updated 👍